src/components/ArtifactSigner/artifact-signer.scss

```diff
@@ -0,0 +1,7 @@
+@import "~@redhat-cloud-services/frontend-components-utilities/styles/variables";
+
+.secure-sign-component {
+  font-size: $ins-fontSize;
+  margin-right: $ins-margin;
+  display: block;
```

Comment:
I wonder if we could add some padding to the component. The screenshots show it really high and tight in the upper left of the screen.

Reply:
I'll fix that.